helm/starboard-exporter/templates/deployment.yaml

```diff
@@ -63,6 +63,9 @@ spec:
         {{- if .Values.exporter.vulnerabilityReports.targetLabels }}
           - --target-labels={{ .Values.exporter.vulnerabilityReports.targetLabels | join "," }}
         {{- end }}
+        {{- if .Values.exporter.CISKubeBenchReports.targetLabels }}
+          - --report-labels={{ .Values.exporter.vulnerabilityReports.targetLabels | join "," }}
```

Comment:
This still uses the target labels from vulnerabilityreports, which I don't think is intentional.

I think we may need to now differentiate between the reports these flags reference.
```suggestion
          - --cis-detail-report-labels={{ .Values.exporter.CISKubeBenchReports.targetLabels | join "," }}
```